packages/fondue/src/components/RichTextEditor/Plugins/EmojiPlugin/EmojiPicker/EmojiPickerContent.tsx

```diff
@@ -0,0 +1,164 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { memo, useCallback } from 'react';
+import { cn } from '@udecode/cn';
+import { Emoji, EmojiCategoryList, EmojiSettings, GridRow, UseEmojiPickerType } from '@udecode/plate-emoji';
+
+export type EmojiPickerContentProps = Pick<
+    UseEmojiPickerType,
+    | 'i18n'
+    | 'onMouseOver'
+    | 'onSelectEmoji'
+    | 'emojiLibrary'
+    | 'isSearching'
+    | 'searchResult'
+    | 'visibleCategories'
+    | 'refs'
+    | 'settings'
+>;
+
+export type EmojiButtonProps = {
+    index: number;
+    emoji: Emoji;
+    onSelect: (emoji: Emoji) => void;
+    onMouseOver: (emoji?: Emoji) => void;
+};
+
+export type RowOfButtonsProps = Pick<UseEmojiPickerType, 'onMouseOver' | 'onSelectEmoji' | 'emojiLibrary'> & {
+    row: GridRow;
+};
+
+const Button = memo(({ index, emoji, onSelect, onMouseOver }: EmojiButtonProps) => {
+    return (
+        <button
+            type="button"
+            aria-label={emoji?.skins[0]?.native}
+            tabIndex={-1}
+            data-index={index}
+            onClick={() => onSelect(emoji)}
+            onMouseEnter={() => onMouseOver(emoji)}
+            onMouseLeave={() => onMouseOver()}
+            className="tw-group tw-relative tw-flex tw-h-[36px] tw-w-[36px] tw-cursor-pointer tw-items-center tw-justify-center tw-border-none tw-bg-transparent tw-text-2xl tw-leading-none"
```

Comment:
Is "tw-group" required? If so it might be an idea to namespace it given the complexity of the components

Reply:
It is needed, but added some extra scoping to it